lib/rules/media-query-no-invalid/index.cjs

```diff
@@ -140,6 +158,31 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * @param {import('@csstools/media-query-list-parser').MediaQuery|import('@csstools/media-query-list-parser').GeneralEnclosed} node
+ * @param {Parameters<Rule>[1]} secondaryOptions
+ * @returns {boolean}
+ */
+function shouldIgnoreNode(node, secondaryOptions) {
+	if (!secondaryOptions?.ignoreFunctions) return false;
+
+	let ignored = false;
+
+	node.walk(({ node: childNode }) => {
+		if (!cssParserAlgorithms.isFunctionNode(childNode)) {
+			return;
+		}
+
+		ignored = optionsMatches(secondaryOptions, 'ignoreFunctions', childNode.getName());
+
+		if (ignored) {
+			return false; // Stop iteration if an ignored function is found
```

Comment:
question

If it uses [`Container.walk`](https://github.com/postcss/postcss/blob/1cc6ac3bf43a12460d3396e2cb76f14075daea1b/lib/container.js#L309-L313)—which I didn't verify—wouldn't the comment be confusing and/or erroneous?